src/test/resources/application.conf

```diff
@@ -1,13 +1,19 @@
 url {
-    api = "http://localhost:9001/graphql"
-    auth = "http://localhost:9002/auth"
+    api = "aHR0cDovL2xvY2FsaG9zdDo5MDAxL2dyYXBocWw=" # http://localhost:9001/graphql
```

Comment:
Is it worth adding a bit more information to the comment to explain that the value is the base 64 string?

Reply:
I've added in the same comment as for the file-format project.